backend/arm64/emit.mlp

```diff
@@ -1162,17 +1212,51 @@ let fundecl fundecl =
   `	.align	3\n`;
   `	.globl	{emit_symbol fundecl.fun_name}\n`;
   emit_symbol_type emit_symbol fundecl.fun_name "function";
+  (* Dynamic stack checking *)
```

Comment:
This part is commented out in `amd64` version, should we also comment it out for `arm64` ?